packages/terra-tabs/src/_CollapsibleTabs.jsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ResizeObserver from 'resize-observer-polyfill';
+import 'terra-base/lib/baseStyles';
+import TabUtils from './TabUtils';
+import Menu from './_TabMenu';
+import styles from './Tabs.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Key of the current active tab.
+   */
+  activeKey: PropTypes.string.isRequired,
+
+  /**
+   * Index of the current active tab.
+   */
+  activeIndex: PropTypes.number.isRequired,
+
+  /**
+   * Tabs to be displayed in the collapsible tab bar.
+   */
+  children: PropTypes.node.isRequired,
+
+  /**
+   * Tabs style. One of: "modular-centered", "modular-left-aligned", or "structural".
+   */
+  variant: PropTypes.oneOf(['modular-centered', 'modular-left-aligned', 'structural']).isRequired,
```

Comment:
Comment till decision?

Reply:
The value passed to this prop is hardcoded to structural in the `Tabs` component. This way I don't have to remove/comment out every reference to modular tabs.